pom.xml

```diff
@@ -89,6 +131,11 @@
   </dependencies>
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-javadoc-plugin</artifactId>
+        <version>3.2.0</version>
```

Comment:
Is this a change done to use this specific version to solve an issue?
Otherwise, AFAIR, this plugin should be managed by the JBoss parent pom when using the `jboss-release` profile.

Reply:
* jboss-parent is max at version 37
* that version is using javadoc plugin 3.1.1 ( https://github.com/jboss/jboss-parent-pom/blob/47c355caeeb5160f11d8fc2352d11e35a6ee72b0/pom.xml#L111 )
* there's an issue with javadoc plugin and jdk 11  ( https://stackoverflow.com/questions/58836862/jdk-11-and-javadoc ) 
* upgrading the version to 3.2.0 solved the issue

Reply:
It was in the description of the PR

Reply:
Awesome, let's keep as it is.
To me a comment in the line above the version would help to remember about this but it's just my love for comments.